pages/about.page.js

```diff
@@ -78,6 +78,40 @@ function About() {
               </div>
             </div>
             <div className="row">
+            <div className="col-lg-3 col-md-4 col-sm-6">
+                <div className="single-topic text-center mb-30">
+                  <div className="topic-img">
+                    <img
+                      src="/assets/img/hackathons/samsung.png"
+                      alt="Solve for Tommorow"
+                    />
+                    <div className="topic-content-box">
+                      <div className="topic-content">
+                        <h3>
+                          <a href="https://www.samsung.com/in/solvefortomorrow/" target="_blank">Solve for Tommorow</a>
+                        </h3>
+                      </div>
```

Comment:
could you please componentize this?? as we did in course/internship section. As we took data from json file.

Reply:
You want me to keep all the data like src , title etc to json file and then fetch data from there just like internship/course section , right?

Reply:
yes create a json for hackathon too!

Reply:
> yes create a json for hackathon too!

Ok got it .